blockchain_api/artifacts/src/chaincode/opetbot/opetbot.go

```diff
@@ -73,6 +83,10 @@ func (t *OpetCode) loadUser(APIstub shim.ChaincodeStubInterface, userKey string)
     return user, nil
 }
 
+/*
+    It is helper function. It is used to load documet structure from the storage.
```

Comment:
loadDocument is a helper to load the document structure from the storage by docKey.